Dockerfile

```diff
@@ -1,17 +1,20 @@
-FROM        sdurrheimer/alpine-glibc
-MAINTAINER  The Prometheus Authors <prometheus-developers@googlegroups.com>
+FROM ubuntu:14.04
+MAINTAINER D.Ducatel
```

Comment:
This line should not be changed, independent of how the SSL cert problem is going to be fixed. Any Dockerfiles under github.com/prometheus/... should list `The Prometheus Authors <prometheus-developers@googlegroups.com>` as maintainers. The `AUTHORS.md` file is the right place to put info about individuals.